assignments/this.js

```diff
@@ -1,26 +1,63 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. explicit binding - uses apply call and bind- arguments
+* 2. implicit binding - 
```

Comment:
We talked about that during lecture. It refers to the object the method is being called on.